windows/mpconfigport.h

```diff
@@ -134,7 +134,8 @@ typedef long mp_off_t;
 typedef void *machine_ptr_t; // must be of pointer size
 typedef const void *machine_const_ptr_t; // must be of pointer size
 
-#define MP_PLAT_PRINT_STRN(str, len) fwrite(str, 1, len, stdout)
+#include <unistd.h>
+#define MP_PLAT_PRINT_STRN(str, len) do { int ret = write(1, str, len); (void)ret; } while (0)
```

Comment:
It doesn't give any warnings on the mingw-gcc I'm currently using, but I copied it like that from the unix port as it probably has good reasons to do it like that